docs/layouts/shortcodes/endpoint-clarification.html

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
The name "clarification" seems wrong to me.  Anything that needs to be clarified should be explained better in the first place. :smile:  endpoint-description?  endpoint-example?

Reply:
Fixed